web/views/member_view.ex

```diff
@@ -1,3 +1,18 @@
 defmodule Community.MemberView do
   use Community.Web, :view
+  @gravatar_host "https://secure.gravatar.com/avatar/"
+
+  def gravatar(email) do
+    hashed_email = email
+    |> String.downcase
+    |> String.strip
+    |> md5_hash
+
+    @gravatar_host <> hashed_email
```

Comment:
thoughts on making `hashed_email(email)` a private function to avoid intermediate variables? I also think it reads nicely.

```
defp hashed_email(email)
  email
  |> String.downcase
  |> String.strip
  |> md5_hash
end
```